ledger/src/signer/Makefile

```diff
@@ -45,7 +45,9 @@ include $(BOLOS_SDK)/Makefile.defines
 # Main app configuration
 APPNAME = "RSK Sign"
 APPVERSION = 3
-APP_LOAD_PARAMS = --appFlags 0x00 $(COMMON_LOAD_PARAMS)
+# No flags
+APPFLAGS = 0x00
+APP_LOAD_PARAMS = --appFlags $(APPFLAGS) $(COMMON_LOAD_PARAMS)
```

Comment:
Are we actually using `APP_LOAD_PARAMS`? Maybe we could remove it if we're not.

Reply:
Nice catch! removed!